server/bin/pbench-index.py

```diff
@@ -290,9 +289,10 @@ def main(options, name):
                 # Sanity check source tar ball path
                 linksrc_dir = os.path.dirname(tb)
                 linksrc_dirname = os.path.basename(linksrc_dir)
-                assert linksrc_dirname == linksrc, (
-                    "Logic bomb!  tar ball "
-                    "path {} does not contain {}".format(tb, linksrc)
+                assert (
+                    linksrc_dirname == linksrc
+                ), "Logic bomb!  tar ball " "path {} does not contain {}".format(
+                    tb, linksrc
```

Comment:
Consider removing the `" "` from the middle of the line.

And, while you're at that, you might as well remove the `Logic bomb` phrase, as well....

Reply:
I'd rather not open the can of worms to additional tweaks to what `black` did.

Reply:
Ew.  Black's behavior here is arguably a bug:  the de-wrapped strings should be rejoined together -- they shouldn't be left with the closing and re-opening quotes.  (If you want to leave the "Logic bomb" phrase there, I suppose that's a separate [issue](https://github.com/distributed-system-analysis/pbench/issues/2714). 😉)

If you're concerned about opening a can of worms, I recommend you revert the change to the Black version:  since this is not the forward-going branch, this sort of change is kind of inappropriate.